### DISCUSSION TITLE
2024-02-07 Weekly Meeting 145 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#441] [Weekly Meetings]
Weekly Meeting 145 is happening on Wednesday, 7 February, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Agenda

1. Can we vote on the new server?
2. working group check ins - did the backlog meeting happen?
3. demos and discussion

**Comment by indiebio** [comment]
from mrmetaverse:
Apple Headset energy, Tom Fiske newsletter about the apps.
The Polys - XR Guild awards, nominations may still be open! - https://thepolys.com/

**Comment by mrmetaverse** [comment]
Vote passed on new discord! 

vote passed, new discord: https://discord.gg/PY8ThwSCM4

**Comment by indiebio** [comment]
jin - https://app.dework.xyz/omi-group i set this up awhile ago

**Comment by mrmetaverse** [comment]
We are going to take stock of who is available, and let them know how to get involved. We will send an email to our W3c list, and inform the entire Angell discord of the new server. We will kickstart our blogging this way as well. 

The hope is to attract some old supporters back, some new supporters in, and align our comms channels.

**Comment by indiebio** [comment]
Working group updates:
glTF: had a meeting with Khronos, made progress. Alexei had good input.
